assets/javascript/game.js

```diff
@@ -0,0 +1,590 @@
+                    break
+                case 'luke':
+                    $('#heros-area3').append(luke.image)
+                    break
+            }
+        }
+        $('#heros-area2').hide();
+        $('#heros-area3').hide();
+        $('#chose-enemy2').hide();
+        $('#attack-button').hide()
+        $('#attack-button2').show()
+        chooseOpp2(luke);
+        console.log(chooseOpp2);
+            if (chooseChar == false) {
+            chooseOpp2(luke);
+            console.log(chooseOpp2);
+        }
+    });
+
+    // Choose Your Final Battle With The Grand Master SITH Lord (Or Should I Say CYST Lord) [Opponent 3]
+    $('#heros-area3').on('click', "img[id = leia]", function(){
+        $('#heros-area3').hide();
+        $('#chose-enemy3').hide();
+        $('#attack-button2').hide()
+        $('#attack-button3').show()
+        chooseOpp3(leia);
+        console.log(chooseOpp3);
+            if (chooseChar == false) {
+            chooseOpp3(leia);
+            console.log(chooseOpp3);
+        }
+    });
+    $('#heros-area3').on('click', "img[id = yoda]", function(){
+        $('#heros-area3').hide();
+        $('#chose-enemy3').hide();
+        $('#attack-button2').hide()
+        $('#attack-button3').show()
+        chooseOpp3(yoda);
+        console.log(chooseOpp3);
+            if (chooseChar == false) {
+            chooseOpp3(yoda);
+            console.log(chooseOpp3);
+        }
+    });
+
+    $('#heros-area3').on('click', "img[id = rey]", function(){
+        $('#heros-area3').hide();
+        $('#chose-enemy3').hide();
+        $('#attack-button2').hide()
+        $('#attack-button3').show()
+        chooseOpp3(rey);
+        console.log(chooseOpp3);
+            if (chooseChar == false) {
+            chooseOpp3(rey);
+            console.log(chooseOpp3);
+        }
+    });
+    $('#heros-area3').on('click', "img[id = luke]", function(){
+        $('#heros-area3').hide();
+        $('#chose-enemy3').hide();
+        $('#attack-button2').hide()
+        $('#attack-button3').show()
+        chooseOpp3(luke);
+        console.log(chooseOpp3);
+            if (chooseChar == false) {
+            chooseOpp3(luke);
+            console.log(chooseOpp3);
+        }
+    });
+};
+
+startGame()
+
+//"The Chosen One Who Will Bring Balance To The Force" Function
+function chooseChar(char){
+    R2D2.play()
+    chosenPlayer = true; 
+    character = char;
+    console.log(character);
+    $('#player-area').append(char.image);
+    $('#player-name').append(char.name);
+    $('#player-health').append(parseInt(char.health));
+    console.log(parseInt(char.health));
+    $('#player-attack').append(parseInt(char.attack));
+    console.log(parseInt(char.attack));
+    $('#player-attack-back').append(parseInt(char.AttackBack));
+    console.log(parseInt(char.AttackBack));
+
+}
+
+//"The Dark Lord Who Will Try To Lure The Chosen One To The Dark Side" Function
+function chooseOpp(opp)   {
```

Comment:
your chooseOpp functions are all really similar, probably a way to use just one.